### DISCUSSION TITLE
Update 6.11

### DISCUSSION THREAD

**Discussion by TBog** [TBog/TBLauncher#269] [Release]
* new icon
* new default theme
* add gestures
* fix crashes
* add image picker for icons
* add more UI settings
* merge color and opacity settings
* add settings migration (allow old backups that had separate color and opacity to import properly)
* add settings activity to the activity stack

<hr /><em>This discussion was created from the release <a href='https://github.com/TBog/TBLauncher/releases/tag/v6.11'>Update 6.11</a>.</em>

**Comment by ltguillaume** [comment]
Disabling `Show dynamic in resultsShow dynamic in results` does not seem to work: 1) the dynamic shortcuts still show up as results while typing and 2) they show up among the `Untagged` items.

**Reply by ltguillaume** [reply]
@TBog Just checking if you've seen this 🙂

**Reply by TBog** [reply]
I've made [an issue](https://github.com/TBog/TBLauncher/issues/278) so I don't forget again. Thanks for the reminder.
